src/main/webapp/scripts/popular.js

```diff
@@ -0,0 +1,14 @@
+function loadPopular() {
+  loadApi(function() {
+    getTrendingFromYoutubeApi()
+    .then((response) => {
+      for(item of response.items) {
+        // Ready in a later PR: display the item nicely on the page
+        console.log(item);
+      }
+    })
+    .catch((error) => {
+      updateDom(error.message, 'popular-list-container');
```

Comment:
Whenever something goes wrong, its good to also throw the error after you update the DOM. More visible in the console with the red text on what went wrong with possibly more details from the actual error.

Reply:
QUnit tests will fail if throws are not caught, but I did a console.error() so that there is a red message in the console about the error.